pytensor/link/jax/dispatch/nlinalg.py

```diff
@@ -104,18 +104,28 @@ def batched_dot(a, b):
     return batched_dot
 
 
-@jax_funcify.register(MaxAndArgmax)
-def jax_funcify_MaxAndArgmax(op, **kwargs):
+@jax_funcify.register(Max)
+def jax_funcify_Max(op, **kwargs):
     axis = op.axis
 
-    def maxandargmax(x, axis=axis):
+    def max(x, axis=axis):
```

Comment:
This didn't make sense, only x will ever by passed
```suggestion
    def max(x):
```